src/library_browser.js

```diff
@@ -225,6 +225,32 @@ var LibraryBrowser = {
       };
       Module['preloadPlugins'].push(audioPlugin);
 
+#if WASM
+      var wasmPlugin = {};
+      wasmPlugin['asyncWasmLoadPromise'] = new Promise((resolve, reject) => resolve());
```

Comment:
I think `=>` is ES6 notation that currently breaks our JS optimizer. Until we fix that, have to use ES5, sorry...